readme.txt

```diff
@@ -106,52 +106,53 @@ There are opportunities for developers at all levels to contribute. [Learn more
+* We're now retrieving all feature settings in the readable `/settings` endpoint.
 
 **Improved Compatibility**
 
-* Fix notices and warnings when using Jetpack on a server running PHP 7.1. #6118, #5821
-* Contact Form: avoid PHP warnings on servers where `print_r()` is disabled. #6234
-* Photon: avoid PHP notices when using plugins that do srcset CDN replacement. #6126, #6170
-* Sharing: force get sharing options when saved options are incorrect due to site configuration issues. Avoids potential fatal errors. #6122
-* Shortcodes: prefix the new Quiz Shortcode CSS classes to avoid conflicts with other Quiz plugins. #6232
-* Widgets: make sure that all widget class names include prefixes to avoid conflicts with other plugins using similar class names. #6149
+* We've eliminated some notices and warnings when using Jetpack on a server running PHP 7.1 or on servers where `print_r()` is disabled.
+* Photon now avoids PHP notices when your site is using plugins that do srcset CDN replacement.
+* When the sharing options appear to be incorrect due to site configuration issues we force it to retrieve the right options avoiding potential fatal errors.
+* We've added some shortcode css and widget class names prefixes to avoid conflicts with other plugins.
 
 **Bug Fixes**
 
-* Admin Page: fix layout issues in IE11. #6109
-* Admin: fix PHP notice when a site has no posts. #6142
-* Ads: deactivate the module when the site doesn't use a plan that support Jetpack Ads. #6242
-* Custom CSS: fix PHP notice. #6154
-* Custom CSS: fix warning appearing when migrating Custom CSS to Core's new CSS editor. #6158
-* General: fix issue with connection banner's "Dismiss" icon. #6182
-* Likes: Prevent a PHP notice when there is no $post object. #5988
-* Photon: Force only Photon URLs to HTTPS. Custom CDN URLs may use a different protocol. #6083
-* Publicize: respect status of Publicize connections: if a connection is not shared with all users on the site, do not trigger Publicize for the other users. #6098
-* Publicize: make sure Publicize is triggered for scheduled posts. #6186
-* SSO: Ensure that logging in via SSO doesn't stop you authenticating to your site using a third-party application relying on the WordPress.com REST API. #6194
-* Sync: avoid PHP notices with the Shortcodes module. #6115, #6151, #6153
-* Tonesque: avoid PHP warnings when an image doesn't exist. #6068
-* Twitter Widget: add noscrollbar option back. #6131
-* Top Posts Widget: fix image size when using the List Layout. #6140
-* WordPress.com REST API: fix notices and warnings when updating data from the WordPress.com central interface. #6104
+* Some sites were using illegal multibyte characters and failing to sync posts, this has been fixed.
+* IE11 was giving our admin layout a bit of a problem but we rectified that.
+* There were some PHP notices popping up when a site has no posts we got rid of them.
+* The new Jetpack Ads feature will no longer be visible if your plan doesn't support them.

```

Comment:
This isn't true -- the card is still there with the upgrade prompt.  We just made sure it gets deactivated if you cancel or downgrade.